core/src/actors/mod.rs

```diff
@@ -10,6 +10,7 @@ use std::{
 
 mod paths;
 mod refs;
+use crate::prelude::NetworkStatusPort;
```

Comment:
This should be up with the use statements, not between mods and their re-exports